models/event.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const Build = require('./build');
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Pipeline = require('./pipeline');
+const Scm = require('../core/scm');
+const Workflow = require('../config/workflow');
+
+const buildId = Joi.reach(Build.base, 'id');
+
+const MODEL = {
+    id: Joi
+        .string().hex().length(40)
+        .description('Identifier of this build')
+        .example('4b8d9b530d2e5e297b4f470d5b0a6e1310d29c5e'),
+    builds: Joi
```

Comment:
@stjohnjohnson Yeah we were debating about this. What is your thought? 
Having eventId inside builds is more consistent with our current models. Like we don't have a list of builds inside a job. 
Having builds inside event helps the UI. UI can just hits `/pipelineId/events` to get a list of events, and for each event it can get the list of builds to display.